recipe/patches/0033-gh24324.patch

```diff
@@ -1,29 +1,25 @@
-From afddfa8176cf8969055c6319504069b250723d3d Mon Sep 17 00:00:00 2001
-From: Isuru Fernando <isuruf@gmail.com>
-Date: Mon, 25 Jan 2021 03:28:08 -0600
-Subject: [PATCH] Make dyld search work with SYSTEM_VERSION_COMPAT=1
```

Comment:
Please keep the attribution and the comments.

Reply:
ok the workflow i was doing was applying the patch and running `git format-patch` to generate the new patch. I guess that deletes the attribution and comments. Should I just delete whatever header `git format-patch` produces and keep the original header?